src/main/java/org/fogbowcloud/arrebol/core/models/specification/Specification.java

```diff
@@ -96,6 +82,14 @@ public void setContextScript(String contextScript) {
         this.contextScript = contextScript;
     }
 
+    public boolean parseToJsonFile(String jsonDestFilePath) {
+
+        List<Specification> spec = new ArrayList<Specification>();
+        spec.add(this);
+        return SpecificationUtils.parseSpecsToJsonFile(spec, jsonDestFilePath);
```

Comment:
hum ...did not like this. I'd be ok to returning a json string representation of this job.

Reply:
This method was removed. It was unnecessary.